tvb/simulator/plot/tools.py

```diff
@@ -250,10 +259,164 @@ def plot_local_connectivity(cortex, cutoff=None):
     pyplot.legend(("Theoretical", "Typical", "Worst", "Best", "Cutoff"))
 
     # set the linewidth of the first legend object
-    #leg.legendHandles[0].set_linewidth(6.0)
-    #leg.legendHandles[1].set_linewidth(6.0)
-    #leg.legendHandles[2].set_linewidth(6.0)
-    #leg.legendHandles[3].set_linewidth(6.0)
+    # leg.legendHandles[0].set_linewidth(6.0)
+    # leg.legendHandles[1].set_linewidth(6.0)
+    # leg.legendHandles[2].set_linewidth(6.0)
+    # leg.legendHandles[3].set_linewidth(6.0)
+
+def temp_avg_timeseries(TAVG, EEG, sim, out, tt):
```

Comment:
Pls pay attention to the numerous times when I removed from the google doc where you prepared this task, all simulation related code.
Your task is to separate here PLOTTING code, not simulations.

Reply:
fixed